### PR TITLE
 Create a test file for the additional_standard_deduction variable

### LINKED ISSUE #4530: Create a test file for the `additional_standard_deduction` variable
Currently not existent

### PR DESCRIPTION
**Pull request by srilareddy**
Fixes #4530

### CODE REVIEW

File: policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+- name: Aged blind count 1, single tax unit 
+  period: 2018
+
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml:3]: delete extra space here

File: policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  period: 2023
+
+  input:
+    aged_blind_count: 2
+    filing_status: JOINT
+  output:
+    additional_standard_deduction: 3000
+    
+
+
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml:20]: make distance between tests one space

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+- name: Aged blind count 1, single tax unit 
+  period: 2018
+  input:
+    aged_blind_count: 1
+    filing_status: SINGLE
+  output:
+    additional_standard_deduction: 1600
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml:7]:
```suggestion
    additional_standard_deduction: 1_600
```

File: policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- name: Aged or blind count 2, Joint tax unit
+  period: 2023
+
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml:11]: delete sapce

File: policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  input:
+    aged_blind_count: 2
+    filing_status: JOINT
+  output:
+    additional_standard_deduction: 3000
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/irs/income/taxable_income/deductions/additional_standard_deduction/aged_blind.yaml:16]:
```suggestion
    additional_standard_deduction: 3_000
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**